openerp/http.py

```diff
@@ -542,7 +543,39 @@ def dispatch(self):
         if self.jsonp_handler:
             return self.jsonp_handler()
         try:
+            rpc_request = logging.getLogger(__name__ + '.rpc.request')
+            rpc_response = logging.getLogger(__name__ + '.rpc.response')
+            rpc_request_flag = rpc_request.isEnabledFor(logging.DEBUG)
+            rpc_response_flag = rpc_response.isEnabledFor(logging.DEBUG)
+
+            endpoint = self.endpoint.method.__name__
+            model = self.params.get('model')
+            method = self.params.get('method')
+            args = self.params.get('args', [])
+
+            if rpc_request_flag or rpc_response_flag:
+                start_time = time.time()
+                start_rss, start_vms = 0, 0
+                if psutil:
+                    start_rss, start_vms = psutil.Process().get_memory_info()
+                if rpc_request and rpc_response_flag:
+                    rpc_request.debug('%s: %s %s, %s',
+                        endpoint, model, method, pprint.pformat(args))
+
             result = self._call_function(**self.params)
+
+            if rpc_request_flag or rpc_response_flag:
+                end_time = time.time()
+                end_rss, end_vms = 0, 0
+                if psutil:
+                    end_rss, end_vms = psutil.Process().get_memory_info()
```

Comment:
`start_rss` and `end_rss` seem unused, can perhaps be marked as such by being named/prefixed with `_`?